### DISCUSSION TITLE
Looking for a way to control the displayed month on DateRangePicker

### DISCUSSION THREAD

**Discussion by hensansi** [mantinedev/mantine#2986] [Q&A] [answered]
I noticed, from the documentation perspective, that we are able to control the month on the Calendar but not on the DateRangePicker. Both share the initialMonth prop but I want to make the user jump to the endDate on selection. (similar to what antd does)

The general idea is the following:
- The user sees the DateRangePicker with two dates preselected - [today, today + 1 year]
- The user wants to select different days so it clicks to change the today date to tomorrow (and the end date automatically changes to tomorrow + 1 year)
- Problem -> To change the end date by a couple of days the user needs to navigate till the end of the range
- Ideal scenario -> the user upon selecting the startDate would be transported to the endDate (via the month prop).

is there any other way of doing this with the DateRangePicker besides the month prop that I'm not seeing?

Code example:

```
const [value, setValue] = useState<DateRangePickerValue>([new Date(2021, 10, 1), new Date(2022, 10, 1)]);
//const [currentMonth, setCurrentMonth] = useState<DateRangePickerValue>(new Date(2021, 10, 1));
  
<DateRangePicker
  //month={currentMonth}
  label="Book hotel"
  placeholder="Pick dates range"
  value={value}
  onChange={([startDate, pickedEndDate]) => {
    const endDate = pickedEndDate || new Date(startDate.getTime());
    if (!pickedEndDate) {
      endDate.setFullYear(startDate.getFullYear() + 1);
      //setCurrentMonth(endDate);
    }
    setValue([startDate, endDate]);
  }}
/>;
  ```

**Comment by rtivital** [comment, marked as answer]
Currently, this is not supported, in 6.0 date package revamp is planned, so current implementation will not change until 6.0 is out

**Reply by hensansi** [reply]
Thank you, this answer already helps me with the refactoring plans.
